src/app/components/LanguageInfo.svelte

```diff
@@ -1,7 +1,15 @@
 <script lang="ts">
   import InputField from "./InputField.svelte";
   import AutoComplete from "../components/AutoComplete.svelte";
+  import worker from "../spawn-worker";
   export let properties: any; // TODO: I am not sure what type to change it to from any
+  // Any is used here to store temporary dynamic object
+  let tempObj: any = {};
+
+  function onInputValue(event: CustomEvent) {
```

Comment:
Perhaps rename this to `updateMetadata`?

```suggestion
  function updateMetadata(event: CustomEvent) {
```

That way, it's clear what is happening in `<AutoComplete>` and `<InputField>`:

`<InputField on:message={updateMetadata} id="authorName">`